component/file_cache/file_cache.go

```diff
@@ -241,7 +241,11 @@ func (c *FileCache) Configure(_ bool) error {
 	}
 
 	err = config.UnmarshalKey("mount-path", &c.mountPath)
-	if err == nil && c.mountPath == c.tmpPath {
+	if err != nil {
```

Comment:
Please add this into UT.

Reply:
mount-path's value is set in mount.go, which we are not mocking this file currently from the individual components. Currently in tests we only mock individual components. Any suggestion on how to mock this value is appreciated.